### ISSUE TITLE
isValid does not work for delayed

### ISSUE THREAD

**Issue by jstastny** [facebook/mention-bot#200]
When `isValid` method is called from https://github.com/facebook/mention-bot/blob/master/server.js#L405, if passes the `currentData`, which does not contain the data format expected by `isValid`.

I am already trying to fix this and will prepare PR. The validation will most likely have to be split to validation on PR level and validation done on the event level. The delayed function will only call the PR validation as it does not have enough data for the other validation.

**Comment by vjeux** [comment]
Good catch. I'm happy to review a pull request if you can get it working :)
